scout/server/blueprints/variant/controllers.py

```diff
@@ -653,6 +653,12 @@ def variant_acmg(store: MongoAdapter, institute_id: str, case_name: str, variant
         store, variant_obj, institute_id, case_name
     )
 
+    genome_build = str(case_obj.get("genome_build", "37"))
```

Comment:
Not in this specific PR, but perhaps we could have one specific function that is called to set the genome build, because atm we have this code all over the place

Reply:
💯 Quite agree! Is far as I know we have it solved air tight in the current case loading; all of these are just to cater to  legacy. A function of that sort could perhaps be set to check and update the case if need be. Or we make a small script and are done with it after...